docs/tools/CLI/cli-setup/cli-reqs.md

```diff
@@ -84,39 +91,49 @@ $ sudo apt-get install libc6:i386 libncurses5:i386 libstdc++6:i386
+To build project, Mbed CLI needs to know where the compiler is installed. It checks three configuration options in the following order:
 
-1. Mbed CLI config (local first, then global)
-1. Environment variables
-1. System path
+1. Mbed CLI config (local first, then global).
+1. Environment variables.
+1. System path.
+
+<!--the order in the list is the same as the original text, but the content has been changed so that PATH is first rather than last. Why?-->
 
 ### Install compilers in the system PATH
 
-Installing the compilers in the system PATH is most often the easiest method, as the compiler installation process will often place the executables in your system's PATH automatically.
+Installing the compilers in the system PATH is most often the easiest method, as the compiler installation process will often place the executables in your system's PATH automatically.<!--often? what does it depend on?-->
+
+<!--what was wrong with the original text
+
+"In addition to toolchain-specific environment variables, Mbed CLI detects executables that are in your system PATH. This means that if you install a toolchain in the system PATH (different for each OS), Mbed CLI will automatically find the toolchain."
+
+-->
 
 ### Configure compiler location with Mbed CLI
 
-It is necessary to configure the location of the compiler with Mbed CLI if you cannot place the compiler in the system PATH or if you have to use a different version of the compiler. Use the `mbed config` command to set this location:
+If you cannot place the compiler in the system PATH, or if you have to use a different version of the compiler than the one in your system path, you must configure the location of the compiler with Mbed CLI. Use the `mbed config` command to set the compiler's location:
 
 ```
 $ mbed config -G ARM_PATH "C:\Program Files\ARM"
 [mbed] C:\Program Files\ARM now set as global ARM_PATH
 ```
 
-Use the `-G` argument to se the compiler location globally. Omit the `-G` argument to use this compiler location for a specific project.
+The `-G` argument sets the location globally, so that it applies to all projects. You can omit the argument if you are configuring a compiler location for a specific project.
+
+<!--Each toolchain requires a specific directory to be used when setting the location:--> <!--I don't think this sentence improves on the original. What it implies is "each toolchain has one universally correct path no all computers everywhere" which we know isn't true. The original version was "Mbed CLI supports a setting for each toolchain path:" which isn't great, but is far less confusing.-->
 
-Each toolchain requires a specific directory to be used when setting the location:
+Each toolchain has its own setting name, and must have a path to <!--where? some of the example paths are down to `bin`, some are to a different level. what's the rule?-->
```

Comment:
There isn't really much a rule unfortunately. The toolchain paths were not implemented consistently initially so we're left with this inconsistent paths. I tried to make the settings less picky but it turned out to be more work than I had for at the time.